pages/advanced/cache.en-US.md

```diff
@@ -0,0 +1,109 @@
+import Callout from 'nextra-theme-docs/callout'
+
+# Custom Cache
+
+<Callout emoji="⚠️" background="bg-yellow-200 dark:text-gray-800">
```

Comment:
```suggestion
<Callout emoji={<span style={{fontFamily: '"Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"'}}>⚠️</span>}>
```

The `⚠️` emoji doesn't seem to display correctly, I'll have to upgrade Nextra. 😢
Let's try this work around for now.

Reply:
TIL about the emoji support 🤔 💭